app/assets/javascripts/discourse/app/widgets/user-menu.js

```diff
@@ -249,7 +251,26 @@ export default createWidget("user-menu", {
   },
 
   dismissNotifications() {
-    return this.state.markRead();
+    const unreadHighPriorityNotifications = this.currentUser.get(
```

Comment:
This is a nitpick more than anything, sorry I didn't notice yesterday, but we don't need to use ember's `.get` for single properties like this, only chained properties. e.g. `currentUser.get("options.some_other_thing")`